api/src/certification/enrolment/domain/usecases/validate-sessions.js

```diff
@@ -128,9 +127,8 @@ async function _createValidCertificationCandidates({
 
     const { certificationCandidateComplementaryErrors, subscriptions } =
       await sessionsImportValidationService.getValidatedSubscriptionsForMassImport({
-        subscriptionLabels: candidateDTO.subscriptionLabels,
+        subscriptionKeys: candidateDTO.subscriptions,
```

Comment:
En testant fonctionnellement j'avais que des CORE lors de l'inscription, mais c'est causé par cette coquille :)

```suggestion
        subscriptionKeys: candidateDTO.subscriptionKeys,
```

En re testant avec c'est tout bon !